lib/elixir/lib/inspect/algebra.ex

```diff
@@ -42,6 +42,11 @@ defmodule Inspect.Opts do
       is useful when debugging failures and crashes for custom inspect
       implementations
 
+    * `:syntax_colors` - when set to a keyword list of colors the output will
+      be colorized. The keys are types and the values are the colors to use for
+      each type. e.g. `[number: :red, atom: :blue]`. Types can include
+      `:number`, `:atom`, `regex`, `:tuple`, `:map`, `:list`, and `:reset`.
+      Colors can be any `IO.ANSI.ansidata` as accepted by `IO.ANSI.format/1`.
```

Comment:
You can link to the type with `t:IO.ANSI.ansidata/0`. See http://elixir-lang.org/docs/master/elixir/writing-documentation.html#recommendations.